discord_bots/automation_bot/automation_bot_instance.py

```diff
@@ -2252,26 +2308,18 @@ async def action_get_raffle_seed(self, ctx: discord.ext.commands.Context, _, __:
         '''Gets the current raffle seed based on reactions'''
 
         if self._rreact["msg_contents"] is not None:
-            await self.display(ctx, "Current raffle seed is: ```\n{}\n```".format(self._rreact["msg_contents"]))
+            await self.display(ctx, "Current raffle seed is:\n```\n{}\n```".format(self._rreact["msg_contents"]))
         else:
             await self.display(ctx, "No current raffle seed")
 
-    async def action_run_test_raffle(self, ctx: discord.ext.commands.Context, cmd, message: discord.Message):
+    async def action_run_test_raffle(self, ctx: discord.ext.commands.Context, _, __: discord.Message):
         '''Runs a test raffle (does not save results)'''
 
-        commandArgs = message.content[len(config.PREFIX):].strip()
-        dry_run = True
-        if len(commandArgs) > len(cmd) + 1 and "actual" in commandArgs[len(cmd) + 1:].strip():
-            dry_run = False
-
-        if dry_run:
-            await self.display(ctx, "Test raffle results:")
-        else:
-            await self.display(ctx, "Raffle results:")
+        await self.display(ctx, "Test raffle results:")
         raffle_seed = self.get_raffle_seed()
 
         raffle_results = tempfile.mktemp()
-        vote_raffle(raffle_seed, 'redis', raffle_results, dry_run=dry_run)
+        vote_raffle(raffle_seed, 'redis', raffle_results, dry_run=True)
```

Comment:
This extra raffle logic was here so we can actually run a real raffle using this if we need to. I'm fine with removing it since it conflicts with the "run test raffle" vibe, but it was actually useful.

Reply:
I don't remember making that change actually - most of these changes are from months ago and I lost my train of thought since. I'll put this back in.